Floofbot/Modules/Help.cs

```diff
@@ -73,5 +75,74 @@ public async Task HelpCommand()
                 Trash = true
             });
         }
+
+        [Summary("Show help for a specific module")]
+        [Command("help")]
+        public async Task HelpCommand([Summary("module")]string requestedModule)
```

Comment:
Will we ever have a module longer than one word?
Just asking since there's currently no [Remainder] marker right now on the requestedModule param.

Reply:
iirc module names cannot be multiple words - so no

Also this leaves the opportunity for even further commands detailing parameter details or even more sub commands  - would be done in a future pr if appropriate